scripts/zones/Bastok_Markets/IDs.lua

```diff
@@ -66,7 +67,10 @@ zones[tpz.zone.BASTOK_MARKETS] =
         THANK_YOU                    = 8279, -- Thank you...
         RETRIEVE_DIALOG_ID           = 12885, -- You retrieve <item> from the porter moogle's care.
         TURNING_IN_SPARKS            = 14206, -- Ohohoho... Turning in sparks, I see.
+        DO_NOT_POSSESS_ENOUGH        = 14230, -- You do not possess enough <item name> to complete the transaction.
```

Comment:
I seem to remember that the syntax of the comments here are somewhat important. Check in with @wrenffxi to see if this needs to be `<item>`

Reply:
Updated comments based on Wren's feedback.  I _hope_ I didn't break anything!